packages/demoing-storybook/demo/README.md

```diff
@@ -0,0 +1,38 @@
+# Example Element
+
+<style>
+  iframe {
+    width: 100%;
+    border: 1px solid rgba(0,0,0,.1);
+    min-height: 300px;
+  }
+</style>
+
+This example element display some information about...
```

Comment:
?

Reply:
this is just for the demo of storybook and just an example how a README for a component could look like... I think making a nice readme could be a task on it's own... and hopefully after storybook 5.0... there will be some focus on docs mode which may influence how a readme looks...

so don't heed too much about it now 🙈

Reply:
Ah I see, ty for explaining